frontend/www/js/omegaup/components/arena/Clarification.vue

```diff
@@ -72,6 +107,7 @@ export default class ArenaClarificationForm extends Vue {
   isPublic = this.clarification.public;
   message = '';
   selectedResponse = 'yes';
+  anotherAnswer = false;
```

Comment:
```suggestion
  showUpdateAnswer = false;
```

por consistencia con la cadena y para ponerle un nombre "booleanoso" a la variable.

Reply:
Listo.